ballerina/tests/utils.bal

```diff
@@ -112,8 +112,12 @@ isolated function writeWebSocketTextMessage(string document, websocket:Client ws
 }
 
 isolated function validateWebSocketResponse(websocket:Client wsClient,
-                                            json expectedPayload) returns websocket:Error?|error {
+                                            json|string expectedPayload) returns websocket:Error?|error {
```

Comment:
Do we need to add `string` type here?

Reply:
It was added because I can check both invalid error messages (which are in strings) and valid json responses using the same method

Reply:
No, I'll remove it.

Reply:
[`json`](https://ballerina.io/learn/by-example/json-type.html) type can accept `string` value also.